tests/benchmark/bm_basics.cpp

```diff
@@ -207,9 +240,17 @@ BENCHMARK_REGISTER_F(BM_VecSimBasics, TopK_BF)
     ->Unit(benchmark::kMillisecond);
 
 BENCHMARK_REGISTER_F(BM_VecSimBasics, TopK_HNSW)
-    ->Arg(10)
-    ->Arg(100)
-    ->Arg(500)
+    // {ef_runtime, k} (recall that always ef_runtime >= k)
+    ->Args({10, 10})
+    ->Iterations(100)
+    ->Args({200, 10})
+    ->Iterations(100)
+    ->Args({100, 100})
+    ->Iterations(100)
+    ->Args({200, 100})
+    ->Iterations(100)
+    ->Args({500, 500})
+    ->Iterations(100)
```

Comment:
no need to specify the iteration for every args. it will take the last `Iterations` value for all args

Reply:
thanks!